src/arrayMethodSort.js

```diff
@@ -5,8 +5,62 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    function swap(i, j, array) {
```

Comment:
Solution looks too complicated. Let's create a separate function and use it as a default value for our compareFunction. This function should compare your items as strings.
![image](https://github.com/mate-academy/js_array-method-sort/assets/80700247/6b38514d-355f-4cb8-995e-79f079b3ae3f)
![image](https://github.com/mate-academy/js_array-method-sort/assets/80700247/673d1e25-d8d0-4187-a79b-8c705b8d04a9)